doc/build/core/custom_types.rst

```diff
@@ -156,7 +156,7 @@ denormalize::
 
         def process_bind_param(self, value, dialect):
             if value is not None:
-                if not value.tzinfo:
+                if not value.tzinfo and not value.tzinfo.utcoffset(value):
```

Comment:
This requires `is not None` otherwise it will raise for utc dates
`bool(datetime.timedelta(0)) is False`

Reply:
also need or

Reply:
fixed in gerrit